openmicroscopy/openmicroscopy
# Url parser improvement

This PR:

- allows passing more complex view params to top links
- fixes https://trello.com/c/gTnJCAs0/64-bug-no-reverse-for


Testing this PR
- add link like:

 ```
bin/omero config append omero.web.ui.top_links '["Data", {"viewname": "webindex", "query_string": {"experimenter": -1}}, {"title": "Repo"}]'
```
- go to http://host/webclient/login/ and log in, check if `Reverse for '{}' not found.` no longer appears.
- test default as well

cc @will-moore
